test/routes/graphql/Query/organizations.test.ts

```
@@ -0,0 +1,137 @@
+import { beforeEach, describe, expect, it } from "vitest";
+import { vi } from "vitest";
+import { GraphQLContext } from "~/src/graphql/context";
```

Biome (1.9.4)

[error] All these imports are only used as types.

Importing the types with import type ensures that they are removed by the compilers and avoids loading unnecessary modules.
Safe fix: Use import type.


(lint/style/useImportType)